packages/flutter/test/material/theme_defaults_test.dart

```diff
@@ -183,4 +183,91 @@ void main() {
     });
   });
 
+  group('FloatingActionButton', () {
+    const BoxConstraints defaultFABConstraints = BoxConstraints.tightFor(width: 56.0, height: 56.0);
+    const ShapeBorder defaultFABShape = CircleBorder(side: BorderSide(color: Color(0xff000000), width: 0.0, style: BorderStyle.none));
+    const EdgeInsets defaultFABPadding = EdgeInsets.zero;
+
+    testWidgets('theme: ThemeData.light(), enabled: true', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          theme: ThemeData.light(),
+          home: Center(
+              child: FloatingActionButton(
+                onPressed: () { }, // button.enabled == true
+                child: const Icon(Icons.add),
+              )
+          ),
+        ),
+      );
+
+      final RawMaterialButton raw = tester.widget<RawMaterialButton>(find.byType(RawMaterialButton));
+      expect(raw.textStyle.color, const Color(0xffffffff));
+      expect(raw.fillColor, const Color(0xff2196f3));
+      expect(raw.elevation, 6.0);
+      expect(raw.highlightElevation, 8.0);
+      expect(raw.disabledElevation, 0.0);
+      expect(raw.constraints, defaultFABConstraints);
+      expect(raw.padding, defaultFABPadding);
+      expect(raw.shape, defaultFABShape);
+      expect(raw.animationDuration, defaultButtonDuration);
+      expect(raw.materialTapTargetSize, MaterialTapTargetSize.padded);
+
+      // The RawMaterialButton that FloatingActionButton builds handles disabled
```

Comment:
Support for setting a FAB's disabled elevation was added recently. Please update this PR per https://github.com/flutter/flutter/pull/24728

Reply:
Done.

Reply:
Since  #24728 FAB handles the disabled elevation conventionally (in the same way as RaisedButton). 

This comment can be deleted, including the TODO.

Reply:
Done!